modules/olm-building-operator-catalog-image.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-restricted-networks.adoc
+
+[id="olm-building-operator-catalog-image_{context}"]
+= Building an Operator catalog image
+
+Cluster administrators can build a custom Operator catalog image to be used by
+Operator Lifecycle Manager (OLM) and push it to a container image registry that
+supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2].
+In the case of a cluster on a restricted network, this registry can be the
```

Comment:
This sentence is long. Suggest breaking it up, perhaps something like, "For a cluster on a restricted network, this registry can be the internal cluster registry. Alternately, it can be the registry that the cluster has network access to, such as the mirror registry created during the restricted network installation or an on-premise Quay Enterprise registry.

Reply:
:fire: